engine-nextgen/src/main/kotlin/graphql/nadel/enginekt/blueprint/NadelExecutionBlueprint.kt

```diff
@@ -16,18 +16,44 @@ import graphql.schema.GraphQLSchema
 data class NadelOverallExecutionBlueprint(
     val schema: GraphQLSchema,
     val fieldInstructions: Map<FieldCoordinates, List<NadelFieldInstruction>>,
+    val underlyingTypeNameToOverallNameByService: Map<Service, Map<String, String>>,
+    val overAllTypeNameToUnderlyingNameByService: Map<Service, Map<String, String>>,
+    val underlyingTypeNamesByService: Map<Service, Set<String>>,
+    val overallTypeNamesByService: Map<Service, Set<String>>,
```

Comment:
This is the pre-computed Set of underlying and overall types that is 95% of the performance fix